From e044f6af1b2b41bc2212551b4fd6353469cb7263 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 12 Sep 2024 16:29:57 +0100
Subject: [PATCH 1297/1350] dts: bcm2712-rpi: Add four missing GPIOs to 2712D0

It's useful for gpioinfo and pinctrl to distinguish between unused and
absent GPIOs. We use "-" for the former and "" for the latter.
With this convention, gpioinfo shows absent GPIOs as "unnamed", while
pinctrl omits them altogether.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712d0-rpi-5-b.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/arch/arm64/boot/dts/broadcom/bcm2712d0-rpi-5-b.dts
+++ b/arch/arm64/boot/dts/broadcom/bcm2712d0-rpi-5-b.dts
@@ -15,10 +15,10 @@
 		"", // GPIO_007
 		"", // GPIO_008
 		"", // GPIO_009
-		"", // GPIO_010
-		"", // GPIO_011
-		"", // GPIO_012
-		"", // GPIO_013
+		"-", // GPIO_010
+		"-", // GPIO_011
+		"-", // GPIO_012
+		"-", // GPIO_013
 		"PCIE_SDA", // GPIO_014
 		"PCIE_SCL", // GPIO_015
 		"", // GPIO_016
